client/src/components/Footer/Footer.css

```
@@ -0,0 +1,58 @@
+footer {
+    background-color:#B3B3B3;;
+    padding: 20px;
+}
+
+.footer-container {
+    display: flex;
+    flex-direction: column;
+}
```

```suggestion
footer {
    display: flex;
    flex-direction: column;
    gap: 10px; /* or whatever gap you want */
    padding: 20px 20px 40px 20px;
    background-color:#B3B3B3;
}
```
we have a footer and then a div inside of it which are fulfilling the same function and can be combined into one

but maybe there is another reason that im missing

there are some adjustments suggestions here that are relative to the comments below

---

I understand what you mean, I'll commit your suggestion thanks! @bazmurphy